test/commands/positive-batch.test.ts

```diff
@@ -120,10 +120,10 @@ describe('PositiveBatch', () => {
       );
 
       expect(result).toEqual([
-        oasResultSuccessString + '\n',
-        oasResultFailureString + '\n',
-        oasResultMixedResultsString + '\n',
-        oasResultMissingPathString + '\n',
+        `${oasResultSuccessString}\n`,
+        `${oasResultFailureString}\n`,
+        `${oasResultMixedResultsString}\n`,
+        `${oasResultMissingPathString}\n`,
```

Comment:
I'm not sure we need to pass in new line characters into template literals because they already handle line [breaks](https://www.digitalocean.com/community/tutorials/understanding-template-literals-in-javascript). If we need a [return](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#multi-line_strings), template literals will keep track of them like [this](https://replit.com/@coolyank/ComplexBestWeb#index.js).

Is there a reason we're passing these into an array? Can we just pass in one template literal into toEqual? Or maybe just one template literal into the array if it's necessary?

Reply:
We'll cover this in another ticket if needed